pom.xml

```diff
@@ -43,6 +44,10 @@
             <id>confluent</id>
             <url>http://packages.confluent.io/maven/</url>
         </repository>
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
```

Comment:
Hva trenger man dette til?

Reply:
Den trengs for brukernotifikasjon-schemas